docs/website/data/integrations.yaml

```diff
@@ -600,7 +600,20 @@ integrations:
       - https://github.com/StyraInc/opa-kafka-plugin
     inventors:
       - redhat
-
+  
+  ccbr:
+    title: ccbr 
+    description: CCBR is a policy management system project. It uses the policy language Rego to implement the CIS benchmark test of cloud native kubernetes. In addition, it integrates gatekeeper, manages its constraint templates, constraints and policies, and supports policy deployment and audit inspection.
+    software:
+      - ccbr
+    labels:
+      category: ZTA
+      layer: Policy Management
+    code:
+      - https://github.com/niuzhi/ccbr
+    inventors:
+      - niuzhi
```

Comment:
⚡ We'll need an "organization" entry for this, otherwise the template fails. Since niuzhi is you, we could also leave the inventor our -- the repo already is in your organization... If the work is sponsored by your employer, I suppose we could add them....

What do we usually do here, @anderseknert?

Reply:
I'd just leave it out.

Reply:
@srenatus  Thank you for your reply, the latest submission is as follows, please review

https://github.com/open-policy-agent/opa/pull/5087